ext/pdo/pdo_stmt.c

```diff
@@ -1627,6 +1556,10 @@ static PHP_METHOD(PDOStatement, rowCount)
 {
 	PHP_STMT_GET_OBJ;
 
+	if (zend_parse_parameters_none() == FAILURE) {
```

Comment:
@carusogabriel should this check be added here?

Reply:
Yes. But, as PDO uses the new ZPP, we should replace the `if`s with 

```suggestion
	ZEND_PARSE_PARAMETERS_NONE();
```